.github/workflows/build.yml

```
@@ -1,11 +1,31 @@
 name: build and push image
+
 on:
   pull_request:
   push:
     branches:
       - azul
+
 env:
-  REGISTRY: ghcr.io
+  # The fully qualified image name of the image to build. The first component is
+  # the host name of the registry that primarily hosts the image.
+  #
+  azul_docker_bigquery_emulator_image: docker.io/ucscgi/azul-bigquery-emulator
+
+  # An optional prefix for the image name if the image is to be pushed to an
+  # alternate registry instead of the one noted in the image name above.
+  #
+  azul_docker_registry: ""
+
+  # The tag from the upstream repo that this repo is forked from.
```

```suggestion
  # The tag from the upstream repo that this repo is forked from.
```

If this comment were accurate, we would never change this variable since we only ever fork a repository once.